lms/djangoapps/bookmarks/tests/test_api.py

```diff
@@ -17,7 +19,22 @@
 from ..models import Bookmark
 
 
-class BookmarksAPITests(ModuleStoreTestCase):
+class BookmarkApiEventTestMixin(EventTestMixin):
+    """ Mixin for verifying that bookmark api events were emitted during a test. """
+    def setUp(self):
+        super(BookmarkApiEventTestMixin, self).setUp('lms.djangoapps.bookmarks.api.tracker')
+
+    def assert_bookmark_event_emitted(self, event_name, username, usage_key):
+        """ Assert that an event has been emitted. """
+        self.assert_event_emitted(
+            event_name,
+            bookmark_id='{},{}'.format(username, unicode(usage_key)),
```

Comment:
Can this also not accept the `resource_id` instead of requiring construction here?